iocs/PICamIOC/iocBoot/iocPICam/st.cmd

```diff
@@ -29,7 +31,11 @@ asynSetTraceIOMask($(PORT), 0, 2)
 dbLoadRecords("$(ADCORE)/db/ADBase.template", "P=$(PREFIX),R=cam1:,PORT=$(PORT),ADDR=0,TIMEOUT=1")
 dbLoadRecords("$(ADPICAM)/db/PICam.template","P=$(PREFIX),R=cam1:,PORT=$(PORT),ADDR=0,TIMEOUT=1")
 
-# Create a standard arrays plugin, set it to get data from Driver.
+# Create a standard arrays plugin, set it to get dae_path not found.
```

Comment:
This section appears to have garbled output from the command line ("e_path not found").  iocInit() should not be here, it appears below.